themes/edx.org/lms/templates/course_modes/choose.html

```diff
@@ -47,6 +47,12 @@
                 window.location.href = '${ecommerce_payment_page | n, js_escaped_string}?sku=' +
                 encodeURIComponent('${sku | n, js_escaped_string}');
             });
+            // This code was added as part of the LEARNER-1726 experiment.
+            $('.v2 button[name=verified_mode]').click(function(e){
+                e.preventDefault();
+                window.location.href = 'https://ecommerce.edx.org/coupons/redeem/?code=EDXTSV35&sku=' +
```

Comment:
Difficulty testing this is what I'm concerned with. How are you going to validate the change on stage? Why not just inject the URL using some Optimizely JS?